plugins/modules/monitoring/statsd.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: statsd
+short_description: Send metrics to StatsD
+description:
+  - The C(statsd) module sends metrics to StatsD.
+  - For more information, see U(https://statsd-metrics.readthedocs.io/en/latest/).
+  - Supported metric types are C(counter) and C(gauge).
+  - Currently unupported metric types are C(timer), C(set), and C(gaugedelta).
+author: "Mark Mercado (@mamercad)"
+requirements:
+  - statsd
+options:
+  state:
+    type: str
+    description:
+      - State of the check, only C(present) makes sense.
+    choices: ["present"]
+    default: present
+  host:
+    type: str
+    default: localhost
+    description:
+      - StatsD host (hostname or IP) to send metrics to.
+  port:
+    type: int
+    default: 8125
+    description:
+      - The port on C(host) which StatsD is listening on.
+  protocol:
+    type: str
+    default: udp
+    choices: ["udp", "tcp"]
+    description:
+      - The transport protocol to send metrics over.
+  timeout:
+    type: float
+    default: 1.0
+    description:
+      - Sender timeout, only applicable if C(protocol) is C(tcp).
+  metric:
+    type: str
+    required: true
+    description:
+      - The name of the metric.
+  metric_type:
+    type: str
+    required: true
+    choices: ["counter", "gauge"]
+    description:
+      - The type of metric.
+  metric_prefix:
+    type: str
+    description:
+      - The prefix to add to the metric.
+  value:
+    type: int
+    required: true
+    description:
+      - The value of the metric.
+  delta:
+    type: bool
+    required: false
+    description:
+      - If the metric is of type C(gauge), change the value by C(delta).
+'''
+
+EXAMPLES = '''
+- name: Increment the metric my_counter by 1
+  community.general.statsd:
+    host: localhost
+    port: 9125
+    protocol: tcp
+    metric: my_counter
+    metric_type: counter
+    value: 1
+
+- name: Set the gauge my_gauge to 7
+  community.general.statsd:
+    host: localhost
+    port: 9125
+    protocol: tcp
+    metric: my_gauge
+    metric_type: gauge
+    value: 7
+'''
+
+
+from statsd import StatsClient, TCPStatsClient
```

Comment:
You need to use import guards and handle the failed `ImportError` in the main module code (search for `missing_required_lib` in existing modules).